### ISSUE TITLE
Accessibility issues in alt text dialog

### ISSUE THREAD

**Issue by mlettini** [KonnorRogers/rhino-editor#316]
We had a Pika user write-in saying:

> The dialog used to add alt text to images has two buttons that are not announced correctly by my screen reader. The Cancel button appears to be unlabelled, and the Save button is announced as being disabled or grayed out.

I can confirm:

- The Cancel button only has an svg inside it. It needs a `title` like "Close dialog" for screen readers.
- The Submit button has `aria-disabled="true}"` on it, until you type some stuff, then it has `aria-disabled="false}"` on it. I assume the errant `}` in there is the issue?

<img width="703" height="37" alt="Image" src="https://github.com/user-attachments/assets/6a5c1cfd-7aa0-466e-8b6e-36d27055a444" />
